code/game/objects/items/weapons/implants/implant_loyality.dm

```diff
@@ -59,8 +59,8 @@
 
 
 /obj/item/weapon/implantcase/loyalty
-	name = "implant case - 'Loyalty'"
-	desc = "A glass case containing a loyalty implant."
+	name = "implant case - 'mindshield'"
+	desc = "A glass case containing a mindshield implant."
```

Comment:
Just curious, why not capitalise it to be Mindshield for name like the old one was?